jenkins/scripts/common/helpers.sh

```diff
@@ -0,0 +1,36 @@
+# Helper functions and variables. Script should be sourced.
```

Comment:
I don't see this file being sourced anywhere. Am I missing something or it will be used in the future?